Olfeo/olfeo-secure-web-gateway/ingest/parser.yml

```
@@ -0,0 +1,46 @@
+name: olfeo
+pipeline:
+  - name: parse_grok
+    description: ""
+    filter: ""
+    external:
+      name: grok.match
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+        pattern: >-
+          \s*%{IP:source_ip} - %{DATA:user} \[%{DATA:date}\] "%{DATA:method}
+          %{DATA:url} HTTP/%{NUMBER:http_version}" %{NUMBER:response_code} - -
+          %{DATA:bytes} %{GREEDYDATA:type}
+        custom_patterns: {}
+  - name: ecs_fields
+    description: ""
+    filter: ""
+    external: null
+stages:
+  ecs_fields:
+    actions:
+      - set:
+          "@timestamp": "{{parse_grok.message.date | to_rfc3339}}"
```

I suggest adding some additional fields.
```suggestion
          "@timestamp": "{{parse_grok.message.date | to_rfc3339}}"
          observer.vendor: "Olfeo"
          observer.product: "Olfeo Secure Web Gateway"
          event.category: ["web"]
          event.type: ["access"]
```